struct.go

```diff
@@ -0,0 +1,57 @@
+package gofakeit
+
+import (
+	"fmt"
+	"math"
+	"reflect"
+)
+
+// Rand fills in elements of a struct with random data
+// based on the value of `fake` tag of exported elements
+func Rand(v interface{}) {
```

Comment:
Can we bat around a couple of different possible names for this?

Reply:
Definitely yes.
I put this as a placeholder only.

Reply:
@brianvoe how about `Struct(v interface{})`?